CMakeLists.txt

```diff
@@ -68,6 +68,10 @@ endif()
 option(ENABLE_CUBLAS_BACKEND "" OFF)
 option(ENABLE_CURAND_BACKEND "" OFF)
 option(ENABLE_NETLIB_BACKEND "" OFF)
+option(DISABLE_HALF_RUTINES "" OFF)
```

Comment:
Ah here it is... an alternative naming could be `ENABLE_HALF_DATA_TYPE` or similar (inspired by [SYCL-BLAS](https://github.com/codeplaysoftware/sycl-blas/blob/ebf361bf0623d5ec65a7399cfca138cb9cde3477/include/operations/blas_constants.h#L213)..)